.github/workflows/concrete_python_test_macos.yml

```diff
@@ -50,31 +50,31 @@ jobs:
       - name: Prepare build environment
         run: |
           set -e
-          cd $GITHUB_WORKSPACE/frontends/concrete-python
+          cd "${GITHUB_WORKSPACE}"/frontends/concrete-python
```

Comment:
For each `"${GITHUB_WORKSPACE}"` variable, you could use instead `${{ github.workspace }}` which doesn't trigger double-quote warning in actionlint.
In any case, does this fancy `cd` call work fine with double-quote in it ?